README.md

```diff
@@ -1,7 +1,7 @@
 # Subnet Rental Canister
 
 ## Running the Project
-If you want to test the project locally, install `dfx` version 0.15.2 and use the following commands:
+If you want to test the project locally, install `dfx` version 0.20.0 and use the following commands:
```

Comment:
Could we just omit the dfx version to avoid bumping it all the time?

Reply:
Added "or later"